skimage/measure/tests/test_fit.py

```diff
@@ -225,11 +225,20 @@ def test_ellipse_model_estimate_from_data():
                'details: '
                'https://github.com/scikit-image/scikit-image/issues/3091 '
                'https://github.com/scikit-image/scikit-image/issues/2670'))
-def test_ellipse_model_estimate_failers():
+@testing.parametrize('epsilon', 10.**np.arange(-4, -20, -1))
+def test_ellipse_model_estimate_failers(epsilon):
+    # This test is known to return true on 32 bit machines with epsilon = 0
+    # Setting the value of espilon to a small number causes it return true
+    # on 64 machiens too.
+    # Unfortunately, it seems to depend on the version of python that is used
+    # Therefore, test for many cases for different values of epsilon.
+
     # estimate parameters of real data
     model = EllipseModel()
     assert not model.estimate(np.ones((5, 2)))
-    assert not model.estimate(np.array([[50, 80], [51, 81], [52, 80]]))
+    assert not model.estimate(np.array([[50, 80],
+                                        [51, 81],
+                                        [52, 80 + epsilon]]))
```

Comment:
according to the comment, what is the epsilon number (it is not clear from the review scope)

Reply:
sorry, quite often, epsilon is a small number in math. I could add a comment, but this wasn't the right approach anyway, and gave the same results as before.